crates/ide/src/inlay_hints.rs

```diff
@@ -1154,7 +1206,6 @@ struct Test { a: Option<u32>, b: u8 }
 
 fn main() {
     let test = Some(Test { a: Some(3), b: 1 });
-      //^^^^ Option<Test>
```

Comment:
These changes are undesirable, the idea behind this "hiding" is to only hide info if it is redundant. In this case here specifically, only `Some` is mentioned, so the fact that this local is of the type `Option<Test>` is not redundant and as such should always be shown.